elements/lisk-p2p/test/functional/actions/get_peer_list.ts

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { expect } from 'chai';
+import { P2P } from '../../../src/index';
+import { wait } from '../../utils/helpers';
+import {
+	createNetwork,
+	destroyNetwork,
+	NETWORK_START_PORT,
+	NETWORK_PEER_COUNT,
+} from '../../utils/network_setup';
+
+describe('PeerPool actions', () => {
+	let p2pNodeList: ReadonlyArray<P2P> = [];
+
+	afterEach(async () => {
+		await destroyNetwork(p2pNodeList);
+	});
+
+	describe('Get Connected Peers', () => {
```

Comment:
Prefer to use the name of the tested functions to follow our patterns, i.e. `getConnectedPeers()`.